timelock-impl/src/main/java/com/palantir/atlasdb/timelock/auth/config/TimelockAuthConfiguration.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.auth.config;
+
+import java.util.List;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.palantir.atlasdb.timelock.auth.api.Credentials;
+
+@JsonSerialize(as = ImmutableTimelockAuthConfiguration.class)
+@JsonDeserialize(as = ImmutableTimelockAuthConfiguration.class)
+@Value.Immutable
+public interface TimelockAuthConfiguration {
+    List<PrivilegesConfiguration> privileges();
+
+    List<Credentials> credentials();
+
+    @JsonProperty("use-auth")
+    boolean useAuth();
```

Comment:
talked offline, this is *the* break glass, will remove `AuthRequirement.NEVER_REQUIRE`

Reply:
do you want a default here of `true` here? otherwise everyone will have to set `useAuth: true`. It should only really be exposed when we need to break glass imo.